spec/views/layouts/listnav/_load_balancer.html.haml_spec.rb

```diff
@@ -1,6 +1,6 @@
-include QuadiconHelper
-
 describe "layouts/listnav/_load_balancer.html.haml" do
+  helper(QuadiconHelper)
```

Comment:
Yeah. Anything within `describe` or `context` is placed within `RSpec::ExampleGroup::...`